config/spotless/copyright.txt

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright $YEAR Pivotal Software, Inc.
+ * <p>
```

Comment:
Do we still need these paragraph tags? I don't remember why they were added. I know from the basepom default checkstyle rules, that it nags on the trailing whitespace in a multiline comment ("' * '"), but as I see it checkstyle is no more.
Can you give it a local try if these paragraph tags are still needed? (Or can somebody comment why it is/was needed otherwise?)